app/src/main/java/org/sagebionetworks/research/mpower/tracking/TrackingTabFragment.java

```diff
@@ -209,7 +201,7 @@ private void setupActionBar(final @Nullable StudyBurstItem item) {
 //        if (!hasShownStudyBurst && item.getBackgroundSurvey() != null) {
 //            launchRsSurvey(item.getBackgroundSurvey());
 //        }
-        if (!hasShownStudyBurst &&
+        if (!trackingTabViewModel.hasShownStudyBurst &&

```

Comment:
All the changes in this class are just adding prefix "trackingTabViewModel." before the 3 vars moved into that class.